Makefile

```diff
@@ -532,7 +538,7 @@ check-community:
 _check:
 	@$(MAKE) fmt license-verify linter run-unit-tests bin
 
-generate-documentation: generate-go-documentation fmt
+generate-documentation: generate-go-documentation fmt-generated
```

Comment:
Probably it's time to rename this target to 'generate' or something like that because it's related not only to docs now